cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpLoggingInterceptor.java

```diff
@@ -37,6 +42,17 @@ void redactHeader(String header) {
         DELEGATE.redactHeader(header);
     }
 
+    //for unit testing
+    void setLogLevel(Level level) {
+        LOGGER.setLevel(level);
+        setLevel();
+    }
+
+    //for unit testing
+    void setLogCollector(StringBuilder collector) {
+        CWMS_HTTP_LOGGER.setCollector(collector);
```

Comment:
You don't need this, see comment on the unit test

Reply:
removed